assets/docs/vasudevgupta7/models/wav2vec2-robust/1.md

```diff
@@ -0,0 +1,50 @@
+# Module vasudevgupta7/wav2vec2-robust/1
+
+Pre-trained speech model (without any head) from Facebook for Automatic Speech Recognition
+
+<!-- asset-path: https://storage.googleapis.com/gsoc-weights/wav2vec2_robust.tar.gz -->
+<!-- task: audio-stt -->
+<!-- network-architecture: wav2vec2 -->
```

Comment:
@MorganR, these checkpoints have slightly different architectures as compared to [old model](https://tfhub.dev/vasudevgupta7/wav2vec2/1) and this model is having different input signature as well. In this case, what should I keep value of `network-architecture`??

Reply:
If there aren't significant changes then I think it's fine to keep it as it is.

Reply:
Okay, this PR is ready for final review (or possibly merge) then!!